Tools/LyTestTools/ly_test_tools/o3de/asset_processor_config_util.py

```diff
@@ -66,28 +65,13 @@ def enable_platform(config_setreg_path, platform):
     :assert: Assert if the platform is not enabled
     :return: None
     """
-    logger.debug("Enabling platform '{0}' in '{1}'".format(platform, AssetProcessorConfig))
+    logger.debug(f"Enabling platform '{platform}' in '{AssetProcessorConfig}'")
 
     file_location = path.join(config_setreg_path, AssetProcessorConfig)
     with RegistrySettings(file_location) as settings:
         settings.set_key(f'{ASSET_PROCESSOR_SETTINGS_ROOT_KEY}/Platforms/{platform}', 'enabled')
 
 
-def enable_all_platforms(config_setreg_path):
```

Comment:
These utilities tend to be used in other modules. Is this safe to remove? Are there no calls to this function anywhere in the O3DE codebase?

Reply:
This function has an undefined reference SUPPORTED_PLATFORMS so it doesn't even work.